commerce_coordinator/apps/commercetools/clients.py

```diff
@@ -266,6 +269,10 @@ def get_orders(self, customer: CTCustomer, offset=0,
             offset=offset,
             expand=list(expand)
         )
+        logger.info(
+            "[UserOrdersView] [CommercetoolsAPIClient] - Get orders query call finished at %s",
+            datetime.datetime.now()
+        )
 
         return PaginatedResult(values.results, values.total, values.offset)
```

Comment:
@NoyanAziz You should add logs around this block to calculate the time it takes.

Reply:
```suggestion
        // Starting log here
        result =  PaginatedResult(values.results, values.total, values.offset)
        // End log here
        return result
```
Like this?